src/notation/view/widgets/selectnotedialog.cpp

```diff
@@ -234,39 +234,38 @@ void SelectNoteDialog::apply() const
         return;
     }
 
-    EngravingItem* selectedElement = interaction->selection()->element();
+    EngravingItem* selectedElement = contextItem(interaction);
     if (!selectedElement) {
         return;
     }
 
-    FilterElementsOptions options = noteOptions();
+    FilterNotesOptions options = noteOptions();
 
     std::vector<EngravingItem*> elements = notationElements->elements(options);
     if (elements.empty()) {
         return;
     }
+    if (isInSelection()) {
```

Comment:
OK so all this code is the same as that in selectdialog.cpp - but a) it was already like that, I just had to add more duplicate code b) I assume this will be rewritten to use QtQuick/QML at some point.